src/1b_prep_aca.jl

```diff
@@ -96,4 +96,75 @@ region_features[!, :geometry] = Vector{AG.IGeometry}(AG.forceto.(region_features
         src_aca_bathy = nothing
         GC.gc()
     end
+
+    if !isfile(joinpath(ACA_OUTPUT_DIR, "$(reg)_waves_Hs.tif"))
+        target_waves_Hs_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, "Hs", reg)))
+        target_waves_Hs = Raster(target_waves_Hs_path, key=:Hs90, crs=crs(src_bathy), mappedcrs=EPSG(4326), lazy=true)
+
+        # Extend bounds of wave data to match bathymetry if needed
+        if size(src_bathy) !== size(target_waves_Hs)
+            target_waves_Hs = extend(crop(target_waves_Hs; to=src_bathy); to=AG.extent(src_bathy))
+            @assert size(src_bathy) == size(target_waves_Hs)
+
+            replace_missing!(target_waves_Hs, -9999.0)
+        end
+
+        # Hacky workaround:
+        # Due to projection schenanigans, the extents appears to be offset by ~5m.
+        # The extent of the data may also cross two UTM zones, even if the wave data
+        # is well within the bathymetry bounds.
+        # We assume the wave coordinates are incorrect, but the cells are the same,
+        # and move on by copying the bathymetry data structure and replace its values with wave data.
+
+        tmp_Hs = copy(src_bathy)
+
+        # Replace data (important: flip the y-axis!)
+        tmp_Hs.data .= coalesce.(target_waves_Hs.data[:, end:-1:1], -9999.0)
+        target_waves_Hs = tmp_Hs
+
+        # Set to known missing value
+        target_waves_Hs.data[target_waves_Hs.data .< -9999.0] .= -9999.0
+        replace_missing!(target_waves_Hs, -9999.0)
+
+        write(joinpath(ACA_OUTPUT_DIR, "$(reg)_waves_Hs.tif"), target_waves_Hs; force=true)
+
+        target_waves_Hs = nothing
+        GC.gc()
+    end
+
+    if !isfile(joinpath(ACA_OUTPUT_DIR, "$(reg)_waves_Tp.tif"))
+        target_waves_Tp_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, Tp, reg)))
+        target_waves_Tp = Raster(target_waves_Tp_path, key=:Tp90, crs=crs(src_bathy), mappedcrs=EPSG(4326), lazy=true)
+
+        # Extend bounds of wave data to match bathymetry if needed
+        if size(src_bathy) !== size(target_waves_Tp)
+            target_waves_Tp = extend(crop(target_waves_Tp; to=src_bathy); to=AG.extent(src_bathy))
+            @assert size(src_bathy) == size(target_waves_Tp)
+
+            replace_missing!(target_waves_Tp, -9999.0)
+        end
+
+        # Hacky workaround:
```

Comment:
It might be best to document this comment section in the README if it's not already there so we don't have to repeat it a couple times in a and b scripts.